client/src/modules/holidays/modals/holiday.modal.html

```diff
@@ -0,0 +1,60 @@
+<form name="HolidayForm" bh-submit="HolidayModalCtrl.submit(HolidayForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li ng-if="HolidayModalCtrl.isCreating" class="title">
+        <span translate>HOLIDAY.ADD_HOLIDAY</span>
+        <label class="badge badge-success" translate>FORM.LABELS.CREATE</label>
+      </li>
+      <li ng-if="!HolidayModalCtrl.isCreating" class="title">
+        <span translate>HOLIDAY.EDIT_HOLIDAY</span>
+        <label class="badge badge-success" translate>FORM.LABELS.UPDATE</label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+    <div class="form-group">
+      <bh-employee-select
+        employee-id="HolidayModalCtrl.holiday.employee_id"
+        name="employee_id"
+        on-select-callback="HolidayModalCtrl.onSelectEmployee(employee)">
+      </bh-employee-select>
+    </div>
+
+    <div class="modal-body" ng-class="{ 'has-error' : HolidayForm.$submitted && HolidayForm.label.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.DESIGNATION</label>
+      <input name="label" ng-model="HolidayModalCtrl.holiday.label" autocomplete="off" class="form-control" required>
+      <div class="help-block" ng-messages="HolidayForm.label.$error" ng-show="HolidayForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+    
+    <div class="modal-body">
+      <bh-date-interval
+        label="FORM.LABELS.PERIODES"
+        date-id="holiday-date"
+        validation-trigger="HolidayModalCtrl.$submitted"
```

Comment:
I think this should be `HolidayForm.$submitted`.  The controller never has a `$submitted` property set on it.